src/App.tsx

```diff
@@ -14,11 +15,106 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string[];
+};
 
-export default App;
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: ['Jam'],
+  };
+
+  addSelect = (item: string) => {
+    this.setState((state) => ({
```

Comment:
не називай аргумент state, краще currentState 
```suggestion
    this.setState((currentState) => ({
```